index.js

```diff
@@ -247,7 +247,10 @@ function onReady () {
             case consts.eventNames.windowCmdMessageCallback:
             let m = {message: json.message}
             if (typeof json.callbackId !== "undefined") m.callbackId = json.callbackId
-            elements[json.targetID].webContents.send(json.name === consts.eventNames.windowCmdMessageCallback ? consts.eventNames.ipcCmdMessageCallback : consts.eventNames.ipcCmdMessage, m)
+            const targetElement = elements[json.targetID]
+            if (targetElement) {
```

Comment:
Could you one line this?

`if (targetElement) targetElement.webContents.send(json.name === consts.eventNames.windowCmdMessageCallback ? consts.eventNames.ipcCmdMessageCallback : consts.eventNames.ipcCmdMessage, m)`